integration/token/fungible/views/signature.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package views
+
+import (
+	"encoding/json"
+
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/assert"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/kvs"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
+	"github.com/pkg/errors"
+)
+
+// Signature contains the input information to generate composite key
+type Signature struct {
+	ObjectType       string
+	TransactionID    string
+	LongTermIdentity view.Identity
+}
+
+type SignatureView struct {
+	*Signature
+}
+
+func (t *SignatureView) Call(context view.Context) (interface{}, error) {
+
+	//generate compositekey
+	k := kvs.GetService(context)
```

Comment:
shall we add a function to the ttx package that checks this? I fear that some refactoring can break this and it becomes hard to track